src/Ticket.java

```diff
@@ -76,25 +74,21 @@ public void setEventCode(String eventCode) {
         this.eventCode = eventCode;
     }
 
-    public void setStadiumSector(char sector) {
-        if (sector < 'A' || sector > 'C') {
-            throw new IllegalArgumentException("Stadium sector must be between 'A' and 'C'.");
-        }
-        this.stadiumSector = sector;
-    }
-
     //**ability to save ticket price
     public void toSaveTicketPrice(double price) {
         BigDecimal ticketPrice = new BigDecimal(price);
         ticketPrice = ticketPrice.setScale(2, RoundingMode.HALF_UP);
         this.price = ticketPrice;
-        System.out.println("The ticket id: "+ this.id+" price is: " + this.price);
+          }
+
+    public String toStringTicketPrice() {
+       return "The ticket id: " + this.id + " price is: " + this.price;
     }
 
     @Override
     public String toString() {
         Date time = new Date(this.time);
-        Date creationTime = new Date(this.creationTime);
+        Date creationTime = new Date(Ticket.creationTime*1000);
```

Comment:
Why we need it? 
Previous line with this is more suitable

Reply:
Because of  (line 15) private static long creationTime = System.currentTimeMillis()/1000; //Creation time in seconds
When I use  Date creationTime = new Date(this.creationTime); I have incorrect time in console. There should be time in millis.

Reply:
this.creationTime*1000 not the same?

Reply:
The same and better then Ticket.creationTime*1000. I get it.